manila/share/drivers/netapp/dataontap/client/client_cmode_rest.py

```diff
@@ -4471,6 +4472,19 @@ def _configure_nfs(self, nfs_config, svm_id):
         self.send_request(f'/protocols/nfs/services/{svm_id}', 'patch',
                           body=body)
 
+    @na_utils.trace
+    def _disable_xattrs(self, svm_id):
+        """Disable xattrs for NFS v4.2"""
```

Comment:
What happens if nfs 4.2 (or rather 4.1, because 4.2 only gets enabled together with 4.1) is disabled, will this raise an error?

Reply:
https://docs.netapp.com/us-en/ontap/nfs-admin/ontap-support-nfsv42-concept.html#nfs-v4-2-security-labels seems like 4.2 is enabled by deault after ontap 9.9.1. If its disabled, whether error is raised or not, needs to check.